docs/podman-build.1.md

```diff
@@ -212,6 +212,10 @@ Recognized formats include *oci* (OCI image-spec v1.0, the default) and
 Note: You can also override the default format by setting the BUILDAH\_FORMAT
 environment variable.  `export BUILDAH_FORMAT=docker`
 
+**-h, --help**
+
+Print usage statement
+
 **--iidfile** *ImageIDfile*
```

Comment:
Missing an `=` here

Reply:
This was something I was unsure about, since the args of the flags can be input after a space or using an equals. Should I change them all to equals for consistency's sake as well?

Reply:
I'd be in favor of that, sure